contracts/test/TestRollup.sol

```diff
@@ -0,0 +1,54 @@
+pragma solidity ^0.5.15;
+pragma experimental ABIEncoderV2;
+
+import { BatchManager } from "../rollup/BatchManager.sol";
+import { Types } from "../libs/Types.sol";
+import { Logger } from "../Logger.sol";
+import { IDepositManager } from "../DepositManager.sol";
+
+contract MockDepositManager is IDepositManager {
+    event EnqueSubtree(bytes32 subtreeRoot);
+
+    function dequeueToSubmit() external returns (bytes32 subtreeRoot) {
+        return bytes32(0);
+    }
+
+    function reenqueue(bytes32 subtreeRoot) external {
```

Comment:
This biased the gas profiling of rollback because the mocked reenqueue is not as expensive as the real depositManager.